orte/orted/pmix/pmix_server_register_fns.c

```diff
@@ -324,6 +288,43 @@ int orte_pmix_server_register_nspace(orte_job_t *jdata)
         kv->data.integer = pptr->name.vpid;
         opal_list_append(pmap, &kv->super);
 
+        /* local size - same for all on host, but not same across namespace */
+        kv = OBJ_NEW(opal_value_t);
+        kv->key = strdup(OPAL_PMIX_LOCAL_SIZE);
+        kv->type = OPAL_UINT32;
+        kv->data.uint32 = jdata->num_local_procs;
+        opal_list_append(pmap, &kv->super);
+
+        /* pass our node ID - same for all on host, but not same across namespace */
```

Comment:
@dsolt I moved the nodeID the other day (and fixed a typo this morning) to correct a problem reported by a user. You probably need to rebase to pickup that change and resolve the conflict - don't want to pass it twice